#! /bin/sh /usr/share/dpatch/dpatch-run
## debian_from-svn-gio-sharp-destdir.dpatch by Matvey Kozhev <sikon@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add DESTDIR to gio-sharp install hook. (upstream SVN 4010)

@DPATCH@

--- f-spot-0.4.4/Makefile.am	2008-05-29 02:37:22.000000000 +0700
+++ f-spot-0.4.4_/Makefile.am	2008-06-20 20:38:28.000000000 +0700
@@ -89,12 +89,12 @@
 install-data-hook: 
 	$(UPDATE_DESKTOP)
 	if [ "x$(DIR_GIO)" != "x" ] ; then \
-		$(install_sh_DATA) $(top_builddir)/gio-sharp/gio/gio-sharp.dll $(libdir)/f-spot/; \
-		$(install_sh_DATA) $(top_builddir)/gio-sharp/gio/gio-sharp.dll.config $(libdir)/f-spot/; \
+		$(install_sh_DATA) $(top_builddir)/gio-sharp/gio/gio-sharp.dll $(DESTDIR)$(libdir)/f-spot/; \
+		$(install_sh_DATA) $(top_builddir)/gio-sharp/gio/gio-sharp.dll.config $(DESTDIR)$(libdir)/f-spot/; \
 	fi
 
 uninstall-hook: 
 	$(UPDATE_DESKTOP)
 	if [ "x$(DIR_GIO)" != "x" ] ; then \
-		rm -f $(libdir)/f-spot/gio-sharp.dll $(libdir)/f-spot/gio-sharp.dll.config; \
+		rm -f $(DESTDIR)$(libdir)/f-spot/gio-sharp.dll $(DESTDIR)$(libdir)/f-spot/gio-sharp.dll.config; \
 	fi
